docs/ingress.md

```diff
@@ -88,10 +88,20 @@ spec:
            number: 8000
 ```
 
-The above ingress resource will create an ApiDefinition inside Tyk's Gateway. The ApiDefinition will offer path-based
- routing listening on /httpbin. Because the referenced template is `myapideftemplate`, the IngressReconciler will
- retrieve the `myapideftemplate` resource and determine that the ApiDefinition object it creates needs to have standard
- auth enabled.
+The above ingress resource will create an ApiDefinition inside Tyk's Gateway. Because the referenced template is `myapideftemplate`, the IngressReconciler will retrieve the `myapideftemplate` resource and determine that the ApiDefinition object it creates needs to have standard auth enabled.
+The ApiDefinition will offer path-based routing listening on /httpbin. Each path in an Ingress must have its particular path type. Kubernetes offers three types of path types: `ImplementationSpecific`, `Exact`, and `Prefix`. Currently, not all path types are supported. The below table shows the unsupported path types based on the examples in [Kubernetes Ingress documentation](https://kubernetes.io/docs/concepts/services-networking/ingress/#examples).
```

Comment:
@buraksekili  Can you create a sub heading (like Path type) and add this content there so it gets highlighted?
Also can you mention what happens if `ImplementationSpecific` type is mentioned?